source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +305,85 @@ provision the following networking API entities, represented in yaml:
+- name: create a router to send RA messages
+  os_router:
     cloud: "{{ cloud_name }}"
     state: present
-    name: lport1
-    network: "{{ ipv6_network.id }}"
-    fixed_ips:
-      - subnet_id: "{{ ipv6_subnet.id }}"
+    name: router0
+    interfaces:
+      - "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless Dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a router to send RA messages
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+~~~~~
+
+## UI considerations
+
+The subnet ipv6_address_mode parameter should be added to the UI, since
+updates to existing subnets are not possible through the UI, and updates to the
+ipv6_address_mode parameter are not allowed, as per the
+[networking API](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,update-subnet-detail#update-subnet). Until the UI is updated,
+the subnet should be created via ovirt-provider-ovn RESTful API or ansible.
+
+It is outside the scope of this feature to implement the UI changes that allow
+logical routers to be provisioned, and the REST API - or its ansible wrapper
+should be used instead. An ansible example is available
+[here](#configuring-stateless-dhcpv6-on-the-subnet).
+
 ## Testing
 
-Two scenarios are recommended:
-  - single IPv6 subnet
-  - multiple IPv6 subnets connected by router
+Two scenarios are recommended, each with two different configurations, leading
+to four different tests:
+  - single IPv6 subnet using dhcpv6-stateful configuration
+  - single IPv6 subnet using dhcpv6-stateless configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateful configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateless configuration
+
+All four scenarios on EL8 guest VMs require a router, whereas on EL7 guest VMs
+only the dhcpv6-stateless configurations require it.
+
+### Configuration mechanism used for each option
+
+|        Option Name        | dhcpv6-stateful | dhcpv6-stateless |
+|:-------------------------:|:---------------:|:----------------:|
+| DNS recursive name server |        DHCP     |      DHCP        |
+|     Domain Search List    |        DHCP     |      DHCP        |
+|   Network Prefix          |        DHCP     |      RA          |
+|   Network Prefix Length   |        RA       |      RA          |
```

Comment:
Please note that the table was just to show the format, the content has to be verified by you.
E.g. dhcp-staeful might provide an address, instead of the prefix.

Reply:
Right, I had noticed that, but carelessly left it as it is.

I think now it is correct.